android-todo-list/architecture-samples-main/app/src/main/java/com/example/android/architecture/blueprints/todoapp/TodoApplication.kt

```diff
@@ -31,5 +34,18 @@ class TodoApplication : Application() {
     override fun onCreate() {
         super.onCreate()
         if (BuildConfig.DEBUG) Timber.plant(DebugTree())
+
+        val deviceNameAndId = android.os.Build.MODEL + Settings.Secure.getString(
+            contentResolver,
+            Settings.Secure.ANDROID_ID
+        )
+
+        runBlocking {
```

Comment:
I don't think we should use runBlocking here. Please add a Loading state if needed and only render after the SDK is initialized